src/nvim/api/extmark.c

```diff
@@ -340,6 +340,8 @@ Array nvim_buf_get_extmarks(Buffer buffer, Integer ns_id, Object start, Object e
 /// @param opts  Optional parameters.
 ///               - id : id of the extmark to edit.
 ///               - end_line : ending line of the mark, 0-based inclusive.
```

Comment:
Should we remove the `end_line` documentation to encourage new code to use `end_row` instead?